arch/lkl/include/uapi/asm/host_ops.h

```diff
@@ -56,6 +56,13 @@ struct ucontext;
  *
  * @thread_create - create a new thread and run f(arg) in its context; returns a
  * thread handle or 0 if the thread could not be created
+ * @thread_create_host - create a new thread as the result of a fork-like call
```

Comment:
```suggestion
 * @thread_create_host - creates a new thread as the result of a fork-like call
```

Reply:
The other calls here are described in the imperative case, I'm not sure why we'd change this one to indicative.

Reply:
Currently it mixes both: "create a new thread [...] and initialises its registers [...]". Either way is fine, but it should be consistent.